package.json

```diff
@@ -177,5 +181,34 @@
   "repository": {
     "type": "git",
     "url": "https://github.com/gitpoint/git-point"
+  },
+  "i18n": {
+    "sourceLanguage": "en",
+    "sourcePath": [
+      "./src/"
+    ],
+    "ignorePath": [
+      "./node_modules"
+    ],
+    "languages": [
+      "de",
+      "en",
+      "eo",
+      "es",
+      "eu",
+      "fr",
+      "gl",
+      "nl",
+      "pl",
+      "pt",
+      "ptBr",
+      "ru",
+      "tr",
+      "uk",
+      "zhCn",
+      "zhTw"
+    ],
+    "messagePath": "src/locale/languages",
+    "translator": "t"
```

Comment:
Added this configuration array into package.json.

This can be moved into the extraction utility if it's preferred. (it's the only place where it's used)